test/test_conjugate_gradient_poisson_solver.jl

```diff
@@ -291,17 +295,30 @@ end
             end
         end
 
-        @testset "Conjugate gradient solver with ImmersedBoundaryGrid on a regular grid and open boundary conditions [$(typeof(arch))]" begin
-            underlying_grid = underlying_grids["regular grid"]
-            @test_broken test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, DiagonallyDominantPreconditioner())
-            @test_broken test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, FFTBasedPoissonSolver(underlying_grid))
-            @test_broken test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, FourierTridiagonalPoissonSolver(underlying_grid))
+        @testset "Conjugate gradient solver with ImmersedBoundaryGrid, a GridFittedBottom and open boundary conditions [$(typeof(arch))]" begin
+            bottom = GridFittedBottom(-0.6)
+            for (underlying_grid_name, underlying_grid) in underlying_grids
+                @info "  Testing $underlying_grid_name with different bottom types and open boundary conditions [$(typeof(arch))]..."
+                @test test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, DiagonallyDominantPreconditioner(), bottom)
+                @test test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, fft_poisson_solver(underlying_grid), bottom)
+            end
         end
 
-        @testset "Conjugate gradient solver with ImmersedBoundaryGrid on a stretched grid and open boundary conditions [$(typeof(arch))]" begin
-            underlying_grid = underlying_grids["stretched grid"]
-            @test test_conjugate_gradient_with_immersed_boundary_grid_and_open_boundaries(underlying_grid, DiagonallyDominantPreconditioner())
```

Comment:
these function names are too long --- this is not blocking, but if you can improve please do